pkg/codec/config.go

```diff
@@ -72,9 +73,9 @@ func (m *ModifiersConfig) ToModifier(onChainHooks ...mapstructure.DecodeHookFunc
 type ModifierType string
 
 const (
-	RenameModifier             ModifierType = "rename"
-	DropModifier               ModifierType = "drop"
-	HardCodeModifier           ModifierType = "hard code"
+	RenameModifierType         ModifierType = "rename"
+	DropModifierType           ModifierType = "drop"
+	HardCodeModifierType       ModifierType = "hard code"
 	ExtractElementModifierType ModifierType = "extract element"
 	EpochToTimeModifierType    ModifierType = "epoch to time"
```

Comment:
Alternatively, this shorter style is sometimes used when it's a prefix :shrug: 
```suggestion
	ModifierRename         ModifierType = "rename"
	ModifierDrop           ModifierType = "drop"
	ModifierHardCode       ModifierType = "hard code"
	ModifierExtractElement ModifierType = "extract element"
	ModifierEpochToTime    ModifierType = "epoch to time"
```